aws-lc-rs/tests/data/ecdsa_sign_asn1_sha3_tests.txt

```diff
@@ -32,3 +32,128 @@ Msg = b3c63e5f5a21c4bfe3dbc644354d9a949186d6a9e1dd873828782aa6a0f1df2f64114a430b
 d = 01a4d2623a7d59c55f408331ba8d1523b94d6bf8ac83375ceb57a2b395a5bcf977cfc16234d4a97d6f6ee25a99aa5bff15ff535891bcb7ae849a583e01ac49e0e9b6
 Q = 04004d5c8afee038984d2ea96681ec0dccb6b52dfa4ee2e2a77a23c8cf43ef19905a34d6f5d8c5cf0981ed804d89d175b17d1a63522ceb1e785c0f5a1d2f3d15e513520014368b8e746807b2b68f3615cd78d761a464ddd7918fc8df51d225962fdf1e3dc243e265100ff0ec133359e332e44dd49afd8e5f38fe86133573432d33c02fa0a3
 Sig = 308188024200ed6d28d082004a5d4259ff72baebd4b555ff9129ee66e8f82a062c50e4b9b0e006b4bd4173b32c724dcd507c012b21da32155c109b5e24d15c117cf1303897809c0242015eae6269c250de32e85fd2644f1b6db29a318e21498216bb779fc022c63165ed75ec19bf8d5af2aa3885b790423f08a9271b5614b1539d29f4ca7139ff3863222a
+
+# [secp256k1,SHA3-256]
+
+# Test vectors for secp256k1 were generated by a modified version of the following go script:
```

Comment:
Don't think we need the entire patch here, but it might help to clarify how it was modified for future maintainability

Reply:
Added description of changes